docs/reference/modules/discovery/ec2.asciidoc

```diff
@@ -1,4 +0,0 @@
-[[modules-discovery-ec2]]
```

Comment:
Ditto adding to https://www.elastic.co/guide/en/elasticsearch/reference/master/redirects.html

Reply:
TIL, thanks. Done in 400b2e46d68.